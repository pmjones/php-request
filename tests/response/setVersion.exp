bool(true)
string(3) "1.1"