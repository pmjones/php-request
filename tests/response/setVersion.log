
---- EXPECTED OUTPUT
bool(true)
string(3) "1.1"
---- ACTUAL OUTPUT
bool(false)
string(3) "1.1"
---- FAILED
