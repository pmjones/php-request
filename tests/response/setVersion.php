<?php
$response = new SapiResponse();
var_dump($response->setVersion('1.1') === $response);
var_dump($response->getVersion());
