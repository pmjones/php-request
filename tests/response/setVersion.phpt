--TEST--
SapiResponse::setVersion
--FILE--
<?php
$response = new SapiResponse();
var_dump($response->setVersion('1.1') === $response);
var_dump($response->getVersion());
--EXPECT--
bool(true)
string(3) "1.1"
