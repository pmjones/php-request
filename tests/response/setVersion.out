bool(false)
string(3) "1.1"